tensorflow/contrib/makefile/Makefile

```diff
@@ -66,7 +66,8 @@ HOST_LIBS := \
 -lstdc++ \
 -lprotobuf \
 -lpthread \
--lm
+-lm \
+-lz
```

Comment:
This section is for the host part of the cross-compilation (which we only do to create the proto_text tool). Since this isn't a 'special' build, and zlib is available on all systems that can handle compilation, I didn't want to add the IS_SLIM_BUILD flag here.